src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/IdentifiersShouldHaveCorrectSuffix.cs

```diff
@@ -179,7 +179,7 @@ private static void AnalyzeCompilationStart(CompilationStartAnalysisContext cont
                 if (userMap.TryGetValue(type.OriginalDefinition, out var suffix) &&
                     !string.IsNullOrWhiteSpace(suffix))
                 {
-                    suffixInfo = SuffixInfo.Create(suffix, canSuffixBeCollection: false);
+                    suffixInfo = SuffixInfo.Create(suffix!, canSuffixBeCollection: false);
```

Comment:
The `string.IsNullOrWhiteSpace` is not taken into account.

Reply:
❕  Use `RoslynString.IsNullOrWhiteSpace` instead of a suppression.

Reply:
Thank you @sharwell